apps/user-office-frontend/src/components/review/ProposalTableReviewer.tsx

```diff
@@ -411,6 +411,13 @@ const ProposalTableReviewer = ({ confirm }: { confirm: WithConfirmType }) => {
         <ProposalReviewContent
           reviewId={urlQueryParams.reviewModal}
           tabNames={reviewerProposalReviewTabs}
+          sepId={
```

Comment:
Question: I understand that this is not part of this PR, but what are your thoughts on why sepId can be both `null` and `undefined`? Is there a conceptual difference between the two or do you think we should strive to use just one in cases like these?

Reply:
As we dont have `strictNullChecks` enabled in the frontend config, `null` and `undefined` are ignored by the language. It is always optimal to use one for consistency until or unless necessary. There are instances such as [**this**](https://github.com/UserOfficeProject/user-office-core/pull/315/files#diff-925a6f70af4c7667d840a9341626e39db2f75ba5ab7920945528541156afe81aR137) where type assertion has to be used because for example in this case `sepId` couldnt be left `undefined`.